README.md

```diff
@@ -0,0 +1,47 @@
+# icinga2-client
+
+This role sets up a small Icinga 2 installation for remote monitoring.
+The zones are automatically configured (parent zones + a global template zone).
+Your nodename is automatically set to the inventory hostname.
+
+A certificate is automatically requested and installed.
+
+This role belongs to [icinga2-master](https://github.com/stuvusIT/icinga2-master).
+
+## Requirements
+
+Ubuntu
+
+## Role Variables
+
+| Name                            | Default/Required          | Description                                                             |
+|---------------------------------|:-------------------------:|-------------------------------------------------------------------------|
+| `icinga2_parent_zone`           | :heavy_check_mark:        | Name of the group of the parent zone                                    |
+| `icinga2_ca_host`               | :heavy_check_mark:        | The host with the CA certificates, a certificate will be requested here |
```

Comment:
Could you elaborate on how ca certificates are requested (over ssh, using an icinga tool ..?)